File: game-backend/src/main/java/ru/arkanoid/gamebackend/models/LevelModel.java
package ru.arkanoid.gamebackend.models;

import lombok.Builder;
import lombok.Data;
import ru.arkanoid.gamebackend.engine.Scale;
import ru.arkanoid.gamebackend.engine.Vector;

@Data
@Builder
public class LevelModel {
    private Vector[] boxesPositions;
    private Scale[] boxesScales;
    private String[] boxesIds;
    private String[] boxesTypes;
    private String[] boxesColors;
    private Vector platformPosition;
    private Vector ballPosition;
    private Scale platformScale;
    private float ballRadius;
    private float width;
    private float height;
}

File: game/Assets/Scripts/Network/Models/LevelModel.cs
using Newtonsoft.Json;

namespace Network.Models
{
    public class LevelModel
    {
        [JsonProperty("boxesPositions")] public PositionModel[] boxesPositions;
        [JsonProperty("boxesScales")] public ScaleModel[] boxesScales;
        [JsonProperty("boxesTypes")] public string[] boxesTypes;
        [JsonProperty("boxesColors")] public string[] boxesColors;
        [JsonProperty("boxesIds")] public string[] boxesIds;
        [JsonProperty("platformPosition")] public PositionModel platformPosition;
        [JsonProperty("ballPosition")] public PositionModel ballPosition;
        [JsonProperty("platformScale")] public ScaleModel platformScale;
        [JsonProperty("ballRadius")] public float ballRadius;
        [JsonProperty("width")] public float width;
        [JsonProperty("height")] public float height;

        public int GetBoxIndex(string id)
        {
            for (int i = 0; i < boxesIds.Length; i++)
            {
                if (boxesIds[i].Equals(id))
                {
                    return i;
                }
            }

            return -1;
        }
    }
}
